vllm_spyre/v1/worker/spyre_model_runner.py

```diff
@@ -2238,11 +2338,20 @@ def update_states(self, scheduler_output: SchedulerOutput):
             # input batch, and update states try to access it there.
             req_id = cached_reqs.req_ids[0]
             req_state = self.requests[req_id]
+            assert isinstance(req_state, ChunkedPrefillRequestState)
             num_computed_tokens = cached_reqs.num_computed_tokens[0]
             if num_computed_tokens < len(req_state.prompt_token_ids):
                 # For now, if it is prefilling, we only need to update num of
                 # computed tokens of the request
                 req_state.num_computed_tokens = num_computed_tokens
+                if self.enable_prefix_caching:
+                    num_cached_blocks = self.kv_cache_manager.\
+                        num_cached_block[req_id]
+                    if num_computed_tokens > \
+                        num_cached_blocks * self.block_size:
+                        # otherwise we're in a dummy iteration
```

Comment:
I find the comment here a bit confusing, here it is not a dummy iteration right? we cache the blocks that were just computed. I would use a comment that describe the current case (caching blocks that were just computed during prefills), not the alternative case (the dummy iteration)